src/Microsoft.ML.AutoML/CodeGen/estimator-schema.json

```diff
@@ -71,7 +71,8 @@
               "DnnFeaturizerImage",
               "Naive",
               "ForecastBySsa",
-              "TextClassifcation"
+              "TextClassifcation",
```

Comment:
are you still use incorrect spelling because of this ?https://github.com/dotnet/machinelearning/blob/1a83389bbd9c2f3ae27de30d10209ca9321adb7d/src/Microsoft.ML.AutoML/CodeGen/trainer-estimators.json#L515